### ISSUE TITLE
survey feedback

### ISSUE THREAD

**Issue by kimby77** [drnikki/diversity-of-the-web#57]
Hello! @nstielau shared this survey with me and asked me to open an issue if I had any thoughts.  I have a few:

1) My biggest piece of feedback is that I think this initiative would benefit from getting clearer on goals and what you want to learn.  This will help inform the types of questions you ask.  Goals of "10,000 unique responses" and so on are great tactical objectives, but there's nothing speaking to what you hope to learn out of this.    These last set of comparison items are more like goals:

_Does employee retention/tenure correlate to diversity
Does management's activity in diversity correlate with employee retention?
Does having a strong mission correct with increased diversity?_

If those are goals of the survey, then we should be asking questions like how long they've been at their current company, what are the management's goals for diversity at their company and do they feel their company is taking the right steps to meet those goals?  

If you go in that direction, I'm happy to work with you on the specifics.

2) Someone mentioned adding questions about whether they are in a leadership role at their company. I 👍 this because that's well known to be where a gap is.  If you ask that question, then the position question should just ask about their role domain and not whether they lead people (so nix the "technical lead" response option), and create a separate set of question(s) tailored to learning about the leadership role.

Example question would be:

Role in your company:
[] Individual Contributor
[] Lead
[] Manager
[] Director
[] VP
[] C-Level

3) Position question: Designer & UX are too vague and similar enough that if I were taking this survey, as a product designer, I wouldn't know which one to check. This is a challenge as these roles are defined differently depending on the company.  

I suggest only having "Designer" as the option.

I listed the breakdown for design roles as I would suggest it though, for your info and if you want to consider more granularity for these roles:
- Graphic Designer/Visual Designer  [am I communicating the right message & vision through the graphical elements, such as typography, iconography, and animation?]
- User Interface/Interaction Designer  [can the user understand immediately how to use this interface?]
- User Experience Designer [how does the entire experience a user has with the product & company feel?]
- Product Designer  [are we building the right solution to a problem? + involved with the overall product development process]
- Researcher [who are our users, what do they need?]

4) Type of company question:  I'm not sure what the differences are between these two:
[] Web Agency / Development Shop
[] Digital Agency

**Comment by drnikko** [comment]
@kimby77 Thanks so much for the feedback!  Working from the bottom up:
1. I think for now, we're going to leave it as two because the idea is that organizations in those buckets will care about the distinction.  
2. There are granularities to each role we've provided (like Technical) and the goal isn't to get a perfect fit to the person's job title, but to give loose categorizations.    I know a lot of UX people who wouldn't check a 'designer' box and vice versa, which is why there are two options there.  

It's not perfect for sure, though.
1.  We added a leadership/role question in #47 
2. That's a great point.  @nstielau updated our goals recently.  I think right now the goal is just to get a broader picture of the people making the web.  We don't, yet, have any hypotheses to prove or support.
